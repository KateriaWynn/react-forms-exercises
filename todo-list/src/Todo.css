.container {
  display: flex;
  justify-content: center;
}

p,
label {
  color: white;
  font-weight: bold;
  width: 60%;
}
