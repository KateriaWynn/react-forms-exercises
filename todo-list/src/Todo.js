import React from 'react';
import './Todo.css';

function Todo({ id, task, handleRemove }) {
  const remove = () => handleRemove(id);
  return (
    <>
      <div className="container">
        <p>{task}</p>
        <button onClick={remove}>X</button>
      </div>
    </>
  );
}

export default Todo;
