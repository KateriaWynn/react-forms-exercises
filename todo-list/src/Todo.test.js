import React from 'react';
import { render } from '@testing-library/react';
import Todo from './Todo';

// smoke test
test('if Todo renders without crashing', () => {
  render(<Todo />);
});

// snapshot test
test('if Todo matches snapshot', () => {
  const { asFragment } = render(<Todo />);
  expect(asFragment()).toMatchSnapshot();
});
