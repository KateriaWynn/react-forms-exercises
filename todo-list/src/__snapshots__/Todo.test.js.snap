// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`if Todo matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <p />
    <button>
      X
    </button>
  </div>
</DocumentFragment>
`;
